web/Views/Landlord/_RegisterPagePartial.cshtml

```diff
@@ -13,121 +27,173 @@
+            </div>
+            <div class="my-3">
+                <label asp-for="Address.County" class="form-label">County</label>
+                <input asp-for="Address.County" class="form-control" required/>
+                <span asp-validation-for="Address.County" class="text-danger"></span>
+            </div>
+            <div class="my-3">
+                <label asp-for="Address.Postcode" class="form-label">Postcode</label>
+                <input asp-for="Address.Postcode" class="form-control" required/>
+                <span asp-validation-for="Address.Postcode" class="text-danger"></span>
+            </div>
+        </div>
     </div>
-    <div class="form-group my-3">
-        <label asp-for="Phone" class="form-label"></label>
-        <input type="tel" asp-for="Phone" class="form-control"/>
-        <span asp-validation-for="Phone" class="text-danger"></span>
+
+    <div class="row mt-3">
+        <div class="col-auto">
+            <div class="my-3">
+                <label asp-for="Email" class="form-label"></label>
+                <input type="email" asp-for="Email" class="form-control"/>
+                <span asp-validation-for="Email" class="text-danger"></span>
+            </div>
+        </div>
+        <div class="col-auto">
+            <div class="my-3">
+                <label asp-for="Phone" class="form-label"></label>
+                <input type="tel" asp-for="Phone" class="form-control"/>
+                <span asp-validation-for="Phone" class="text-danger"></span>
+            </div>
+        </div>
     </div>
 </div>
 
-
 <div class="registerTab" id="Landlord details">
+    <h2 class="font-h3 mb-5">@(!Model.Unassigned ? "Let us get to know you better" : "Tell us a bit about the landlord")</h2>
+
     <div class="row">
-        <div class="col-md-5">
-            <div class="form-group my-3">
+        <div class="col-auto">
+            <div class="my-3">
                 <label asp-for="LandlordType" class="form-label"></label>
                 <select asp-for="LandlordType" class="form-select" required>
-                    <option selected disabled value="">Landlord Type</option>
+                    <option selected disabled value="">Choose a type</option>
                     <option value="Charity">Charity</option>
                     <option value="Private Residential Sector">Private Residential Sector</option>
                     <option value="Registered Provider">Registered Provider</option>
                     <option value="Local Authority">Local Authority</option>
                 </select>
                 <span asp-validation-for="LandlordType" class="text-danger"></span>
             </div>
-            <div class="form-group my-3">
+            <div class="my-3">
                 <label asp-for="IsLandlordForProfit" class="form-label"></label>
                 <select asp-for="IsLandlordForProfit" class="form-select" required>

```

Comment:
Should we use the Html DropdownListFors or are we not changing any functionality here?

Reply:
I was focusing on just styling the page, but looking at it I think the tag helpers (with the `asp-for` attributes) mean that these dropdowns remain filled in if validation fails. Unless there's another reason to use `DropdownListFor` over tag helpers?